### PR TITLE
Add metrics to QC

### PR DESCRIPTION
**Pull request by DarianHole**
Added qc metrics:
- Total number of N's in the consensus sequence
- Coverage Depth

### CODE REVIEW

File: bin/qc.py [hunk 1]
```diff
@@ -142,9 +151,11 @@ def go(args):
 
 
     qc_line = { 'sample_name' : args.sample,
+                    'count_N' : count_N,
                 'pct_N_bases' : "{:.2f}".format(pct_N_bases),
           'pct_covered_bases' : "{:.2f}".format(pct_covered_bases), 
            'longest_no_N_run' : largest_N_gap,
+             'depth_coverage' : "{:.2f}".format(depth_coverage),
```
Comment (Takadonet) [inline bin/qc.py:158]: Round up to next integer for depth of coverage.

File: bin/qc.py [hunk 2]
```diff
@@ -59,6 +59,13 @@ def get_covered_pos(pos_depth, min_depth):
     
     return counter
 
+def get_depth_coverage(pos_depth, ref_length):
```
Comment (Takadonet) [inline bin/qc.py:62]: does pos_depth  contain every position in the reference or just the ones with at least one read. Please double check.

Comment (DarianHole) [reply bin/qc.py:62]: I believe it does. It creates a list that is [id, position, coverage] and has a length of 29903 (which matches the len of the ref sequence) when testing it again here

Comment (Takadonet) [reply bin/qc.py:62]: Perfect. Thanks for checking

File: bin/qc.py (hunk 1 above)
Comment (DarianHole) [reply bin/qc.py]: Done with math.ceil (if thats not ok I can do it with another solution)

**Pull request merged**